### ISSUE TITLE
background-image image-set url issue

### ISSUE THREAD

**Issue by AlleeX** [jakubpawlowicz/clean-css#910]
before

```
.style {
  background-image: image-set(
    url("//s1.server.com/img.png") 1x,
    url("//s1.server.com/img@2x.png") 2x
  );
}
```

after

```
.style{background-image:image-set(
    url("//s1server.com/img.png") 1x,
    url("//s1server.com/img@2x.png") 2x
  )}
```

"//s1.server.com/img.png"  ---> "//s1server.com/img.png"

**Comment by AlleeX** [comment]
![selection_204](https://cloud.githubusercontent.com/assets/10462372/23793527/67c7f54e-0594-11e7-838a-b34f497b0662.png)

**Comment by mAiNiNfEcTiOn** [comment]
Yup, that's a good one @AlleeX ... Using `image-set` on `background-image` causes an issue with the URLs, since the value of the property is not a url, but a function that uses urls.

Anyway, downgrading to version 3.x fixes it.

Given I am using `gulp-clean-css` I had also to downgrade to v2.2.2 of that project.

With 3.4.25:

<img width="1136" alt="screen shot 2017-03-11 at 09 33 11" src="https://cloud.githubusercontent.com/assets/1002056/23821801/e395f15e-063d-11e7-88a0-70d199277d5d.png">

With 4.0.8:

<img width="716" alt="screen shot 2017-03-11 at 09 34 43" src="https://cloud.githubusercontent.com/assets/1002056/23821807/ff43dd62-063d-11e7-8644-14c49446a51e.png">

**Comment by jakubpawlowicz** [comment]
Oops, sorry about this. Fix is coming!

**Comment by mAiNiNfEcTiOn** [comment]
Thanks for the fast response @jakubpawlowicz 👍 

Keep up the awesome work ;-)

**Comment by jakubpawlowicz** [comment]
Thanks @mAiNiNfEcTiOn! This issue is fixed in 4.0.9.
